### ISSUE TITLE
New release 5.0.0

### ISSUE THREAD

**Issue by maethu** [collective/collective.solr#109]
I would really appreciate a new release of collective.solr!

1. We could go ahead with several open PR's
2. I'm able to port more features from ftw.solr to collective.solr.
3. No deployments from master branch anymore :fearful: :stuck_out_tongue_winking_eye: 

Or someone may giving me access on pypi to the repo and I'm gonna make a new release (4.2.0).

\cc   @do3cc @tisto

**Comment by do3cc** [comment]
Your pypi username please :-)

**Comment by tisto** [comment]
I will do a release over the weekend...

**Comment by tisto** [comment]
@maethu I was planing to do a 4.2 release today. Though, I still hesitate because of the atomic update feature. That might be a breaking change that requires a 5.0 release. I need to run at least a few tests with existing instances before I can make a release. Will try to find time tomorrow.

Just to be sure, after this release we need to:

- Update the Solr configuration (in Plone)
- Run a full reindex
- Update the Solr Buildout recipe as well? You amended the Buildout configuration in your commit as well: https://github.com/collective/collective.solr/commit/4384fcf3c85be89bb3bfb1724b382fc984a398b2 If this is true, I would consider this a breaking change and tend to do a 5.0 release.

**Comment by maethu** [comment]
@tisto 

Well, you need to update the solr conf and do a full reindex only if you want to profit from the atomic update feature. IMHO this does not break the compatibility with prev. version (4.1.x).  But you're still right that this requires a 5.0 release. Because Integrators has to do some work on their buildout (version pinnings / change solr config) if they want to gain all features. Just update to the new version is definitely no enough. It's also more likely that people are going to read the new instructions more carefully, because of the major release.

**Comment by tisto** [comment]
@maethu I just tested c.solr master on a project with 5 solr cores, lots of data and about 500 lines of Solr buildout configuration. Had to upgrade c.solr, run the upgrade step, amend buildout configuration (we are still on c.r.solrinstance 5.x) and run a reindex. Looks good so far.

Though, I think we should go though the docs before making a 5.0 release and make sure they are up to date. Maybe we should add some troubleshooting instructions as well, so people do not get frustrated on the upgrade. Will try to find time over the weekend...

**Comment by jone** [comment]
As we have more and more productive deployments with the newest solr (master) it would be great to have a pypi release. It could be a beta release, if the docs are not ready yet.

**Comment by tisto** [comment]
@jone will do a release today!

**Comment by jone** [comment]
@tisto Thanks for the release! ✨

**Comment by maethu** [comment]
💖

**Comment by tomgross** [comment]
I guess this is done: https://pypi.python.org/pypi/collective.solr/5.0

**Comment by maethu** [comment]
@tisto Thanks for the 5.0 release.
It's now deployed on the capital city website of Switzerland -> http://www.bern.ch
With collective.solr as base, we were able to implement a well received search service for this website. 

Examples:
- http://www.bern.ch/@@search?facet=true&facet.field=review_state&facet.field=site_area&facet.field=portal_type&SearchableText=Abfall
- Also try the livesearch (Search for "Abfall")

There are also several new features we've implemented for bern.ch, which are available in ftw.solr. I'm gonna try to port them into collective.solr soon.

Thanks again!!
